README.md

```diff
@@ -18,18 +19,15 @@
 
 ```bash
 rbenv install 3.1.2
-rbenv local 3.1.2
```

Comment:
there is no need to run `rbenv local` because we have the `.ruby-version` file and rbenv takes the ruby version that needs to be used automatically from that file